src/replication.c

```diff
@@ -3565,7 +3776,7 @@ void replicationDiscardCachedMaster(void) {
  * passed as argument as the socket for the new master.
  *
  * This function is called when successfully setup a partial resynchronization
- * so the stream of data that we'll receive will start from were this
+ * so the stream of data that we'll receive will start from where this
```

Comment:
这个回退回去（怕以后说合并 redis 新代码产生冲突），哦哦哦看到这里我猜你本地的代码应该是基于 redis unstable 的？
```suggestion
 * so the stream of data that we'll receive will start from were this
```

Reply:
这个pr有版本冲突，我重新提交一个pr